content.js

```diff
@@ -6,28 +6,25 @@ function debug(message) {
   console.debug(`squashed-merge-message: ${message}`);
 }
 
-function copyPrDescription() {
+async function copyPrDescription() {
   debug('copy PR description');
-  const prTitleEl = document.getElementById('issue_title');
-  if (!prTitleEl) {
-    warn('failed to find PR title element');
-    return;
-  };
 
-  const prNumberMatch = window.location.pathname.match('/pull/(?<pr_number>[0-9]+)$');
-  if (!prNumberMatch) {
-    warn('failed to find match for PR number in URL');
+  const match = window.location.pathname.match('^/(?<repo>[^/]+/[^/]+)/pull/(?<pr_number>[0-9]+)$');
+  if (!match) {
+    warn('failed to find match for repo and PR number in URL');
     return;
   };
+  const repo = match.groups['repo'];
+  const prNumber = match.groups['pr_number'];
 
-  let prBodyEl = document.querySelector('.comment-form-textarea[name="issue[body]"]');
-  if (!prBodyEl) {
-    prBodyEl = document.querySelector('.comment-form-textarea[name="pull_request[body]"]');
-    if (!prBodyEl) {
-      warn('failed to find PR body element');
-      return;
-    };
-  };
+  let prMetadata = await getPrMetadataFromDocument();
+  if (!prMetadata) {
+    prMetadata = await getPrMetadataFromApi();
```

Comment:
```suggestion
    prMetadata = await getPrMetadataFromApi(repo, prNumber);
```

Reply:
I noticed this myself and already committed a fix.  (It took me a few hours to get back to testing everything in that second commit.)